server/authz/policy.rego

```diff
@@ -261,6 +261,21 @@ allow {
 	action == write
 }
 
+# Allow read for host health for global admin/maintainer, team admins, observer, gitops.
+allow {
+	object.type == "host_health"
+  subject.global_role == [admin, maintainer, observer, gitops][_]
```

Comment:
Heads-up, you may want to double-check for `gitops`, normally this is a `write`-only role, it should never have read access.